packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,4 +1,73 @@
 {{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  <div class="hds-pagination-bar__total-count">1–10 of 100</div>
+
+  <nav class="hds-pagination-bar__pagination">
+    <ul class="hds-pagination-bar__page-list">
+      <li class="hds-pagination__page hds-pagination__page--previous">
+        <a class="hds-pagination__page-link" aria-label="Previous page" href="/page-?">
+          <FlightIcon @name="chevron-left" class="hds-pagination__nav-icon" /><span
+            class="hds-pagination__page-label"
+          >Previous</span>
+        </a>
+      </li>
+      <li class="hds-pagination__page hds-pagination__page--current">
+        <a class="hds-pagination__page-link" href="/page-1">1</a>
```

Comment:
If this is the current link for the current page, you'd want to add `aria-current="page"` here.

Reply:
Should I use an anchor link for the current page like I have here now or else remove/replace it with something non-interactive?